packages/@react-aria/ssr/docs/useIsSSR.mdx

```diff
@@ -50,5 +50,5 @@ import {useIsSSR} from '@react-aria/ssr';
 function MyComponent() {
   let isSSR = useIsSSR();
   return <span>{isSSR ? 'Server' : 'Client'}</span>;
-}
+};
```

Comment:
I don't think semicolons usually go after normal function declarations?

Reply:
#2708 fixes this